
syntax = "proto3";
package telir;

enum BuiltinFunc {
  ADD_U32 = 0;
  SUB_U32 = 1;
  MUL_U32 = 2;
  DIV_U32 = 3;
}
